packages/radio-group/test/radio-group.test.js

```diff
@@ -493,10 +493,12 @@ describe('radio-group', () => {
   });
 
   describe('aria-labelledby', () => {
+    let error, helper, label;
+
     beforeEach(() => {
-      group = fixtureSync(`<vaadin-radio-group></vaadin-radio-group>`);
-      helper = group._helperNode;
+      group.helperText = 'Choose one';
```

Comment:
Let's revert it to creating a fixture. Note, each test suite in this file creates fixtures on its own.

```suggestion
      group = fixtureSync(`<vaadin-radio-group helper-text="Choose one"></vaadin-radio-group>`);
```

Reply:
Done.